test/models/graphcast/test_graphcast_snmg.py

```diff
@@ -24,20 +24,23 @@
 import torch
 from graphcast.utils import create_random_input, fix_random_seeds, get_icosphere_path
 from pytest_utils import import_or_fail
-from torch.nn.parallel import DistributedDataParallel
 
-from modulus.distributed import DistributedManager
+from modulus.distributed import DistributedManager, mark_module_as_shared
 from modulus.models.graphcast.graph_cast_net import GraphCastNet
 
 icosphere_path = get_icosphere_path()
 
 
+torch.backends.cuda.matmul.allow_tf32 = False
```

Comment:
Why is this needed here?

Reply:
I sometimes use this to get tighter bounds on checking the numerics of different implementations. TF32 actually can have quite large errors - especially in a model with multiple layers. 
I have to admit that I am not consistent with this, but especially when starting to develop things, one can identify actual implementation errors quicker than having to figure out whether minor differences come from numerics or bugs.